.env.j2

```diff
@@ -0,0 +1,10 @@
+FLASK_APP=todo_app/app
+FLASK_ENV=production
+
+SECRET_KEY={{ secret_key }}
+TRELLO_KEY={{ trello_key }}
+TRELLO_TOKEN={{ trello_token }}
+TRELLO_BOARD_ID=vLezjNr2
+TRELLO_TODO_LIST_ID=6303cc311927f1003a61a208
+TRELLO_DONE_LIST_ID=6303cc33607caa00a20aecc2
+TRELLO_DOING_LIST_ID=63176f1aa6f504001861b4ca

```

Comment:
Not a problem, but worth noting that we probably do want these to be variable between environments - setting them as  such in the playbook doesn't have to make them prompts that users have to enter, but makes it a bit more obvious where to change them:
```
- name: Run todo app
  hosts: webservers
  remote_user: ec2-user
  vars:
    trello_api_board_id: 'vLezjNr2'
    ...
```